Formula/libgpg-error.rb

```diff
@@ -1,8 +1,8 @@
 class LibgpgError < Formula
   desc "Common error values for all GnuPG components"
   homepage "https://www.gnupg.org/related_software/libgpg-error/"
-  url "https://gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.38.tar.bz2"
-  sha256 "d8988275aa69d7149f931c10442e9e34c0242674249e171592b430ff7b3afd02"
+  url "https://gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.39.tar.bz2"
+  sha256 "4a836edcae592094ef1c5a4834908f44986ab2b82e0824a0344b49df8cdb298f"
   license "GPL-2.0"
```

Comment:
```suggestion
  license all_of: ["LGPL-2.1-or-later", "GPL-2.0-or-later"]
```

According to the README:

```
Libgpg-error is free software; you can redistribute it and/or modify
it under the terms of the GNU Lesser General Public License as
published by the Free Software Foundation; either version 2.1 of the
License, or (at your option) any later version.  See the file
COPYING.LIB for copyright and warranty information.  See the file
AUTHORS for a list of authors and important mail addresses.

However, some files (for example src/mkerrnos.awk) used in the build
process of the library and the manual are covered by a different
license.  Please see the header of these files and the file COPYING
for copyright and warranty information on these files.  A special
exception in the copyright license of these files makes sure that the
output in the build process, which is used in libgpg-error, is not
affected by the GPL.
```

If the exceptions are used only in the build process, do we need to list `GPL-2.0`?

Reply:
I'd say we don't have to. @Homebrew/plc opinions?

Reply:
My reading of it is that as long as we don't distribute these files I think it's fine.

Reply:
Makes sense. Looking through the files in the cellar for 1.38, it looks like most are `LGPL-2.1-or-later`. There are a few that indicate that they were automatically created by one of the files that had the `GPL` license but don't indicate that they, themselves, are licensed under `GPL`.

Looking more closely now, I'm seeing this header in a few place:

```python
# Copyright (C) 1999, 2002, 2003 Free Software Foundation, Inc.
#
# This file is free software; as a special exception the author gives
# unlimited permission to copy and/or distribute it, with or without
# modifications, as long as this notice is preserved.
#
# This file is distributed in the hope that it will be useful, but
# WITHOUT ANY WARRANTY, to the extent permitted by law; without even the
# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
# SPDX-License-Identifier: FSFULLR
```

I don't think `FSFULLR` is allowed by Debian

Reply:
So.. how do I proceed here?

Reply:
Well, I will differ to the PLC here but I don't see any indication that there is a choice for whether to use `FSFULLR` or not, so I don't think we can accept it. I'm not sure how we've handled similar situations so far.

Reply:
Does any other distro ship this software? And what license do they claim it has?

Reply:
Looks like MacPorts says `LGPL-2.1-or-later`: https://ports.macports.org/port/libgpg-error/summary

Reply:
Can we just merge without license changes?

Reply:
Yes, let's do that. Should we open an issue for it?